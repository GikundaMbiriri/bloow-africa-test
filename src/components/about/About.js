import React from 'react'

function About() {
  return (
    <div className='w-full'>
    <h1 className='text-2xl font-bold py-3 text-center text-dark-blue'>About Bloow Africa</h1>
    <ul className='list-disc w-2/3 m-auto'>
      <li className='pb-2'>We  aim at developing the best music platform for artists world wide and locally.</li>
      <li className='pb-2'>We  aim at developing the best music platform for artists world wide and locally.</li>
      <li className='pb-2'>We  aim at developing the best music platform for artists world wide and locally.</li>
      <li className='pb-2'>We  aim at developing the best music platform for artists world wide and locally.</li>
      
    </ul>


    </div>
  )
}

export default About