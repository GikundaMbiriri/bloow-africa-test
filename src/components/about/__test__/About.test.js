import React from 'react'
import ReactDOM from 'react-dom'
import {render, screen} from '@testing-library/react'
import renderer from 'react-test-renderer';
import About from '../About'
it("it renders without crashing",()=>{
    const div = document.createElement('div');
    ReactDOM.render(<About></About>,div)
})

it('renders correctly', () => {
    const tree = renderer
      .create(<About></About>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });