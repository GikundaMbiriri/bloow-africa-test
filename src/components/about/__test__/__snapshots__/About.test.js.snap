// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="w-full"
>
  <h1
    className="text-2xl font-bold py-3 text-center text-dark-blue"
  >
    About Bloow Africa
  </h1>
  <ul
    className="list-disc w-2/3 m-auto"
  >
    <li
      className="pb-2"
    >
      We  aim at developing the best music platform for artists world wide and locally.
    </li>
    <li
      className="pb-2"
    >
      We  aim at developing the best music platform for artists world wide and locally.
    </li>
    <li
      className="pb-2"
    >
      We  aim at developing the best music platform for artists world wide and locally.
    </li>
    <li
      className="pb-2"
    >
      We  aim at developing the best music platform for artists world wide and locally.
    </li>
  </ul>
</div>
`;
